modules/control/controller/controller_agent.h

```diff
@@ -26,11 +26,12 @@
 #include <memory>
 #include <vector>
 
-#include "modules/common/util/factory.h"
-#include "modules/control/controller/controller.h"
 #include "modules/control/proto/control_cmd.pb.h"
 #include "modules/control/proto/control_conf.pb.h"
 #include "modules/planning/proto/planning.pb.h"
+
+#include "modules/common/util/factory.h"
+#include "modules/control/controller/controller.h"
 #include "third_party/ros/include/ros/ros.h"
```

Comment:
Following the Google C++ Style here: https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes

ros.h should be included earlier.